server/api/controller/cron.js

```diff
@@ -418,21 +419,35 @@ const fetchPlanStatus = () => {
 
 				return MavatAPI.getPlanStatus(plan).then(async (planStatuses) => {
 					try {
+
+						// cut description to match db field length
+						planStatuses.forEach(ps => {
+							if (Boolean(ps.attributes.status_description) && ps.attributes.status_description.length > 255) {
+								ps.attributes.status_description = ps.attributes.status_description.substr(0, 254);
+							}
+						});
 						const mostRecent = planStatuses.sort((statusA, statusB) => { Date.parse(statusB.attributes.date) - Date.parse(statusA.attributes.date); });
 						const now = moment().format('YYYY-MM-DD HH:mm:ss');
 						Log.debug('updating last_visited_status to:', now );
 
 						if (! mostRecent[0]) {
 							Log.debug('No status in mavat for plan', plan.get('id'));
-							plan.save({ 'last_visited_status': now });
+							await plan.save({ 'last_visited_status': now });
 							return;
 						}
 						const mostRecentStatus = mostRecent[0].attributes.status;
 						Log.debug('updating plan status to:', mostRecentStatus);
-						plan.save({ 'last_visited_status': now , 'status': mostRecentStatus });
+						await plan.save({ 'last_visited_status': now , 'status': mostRecentStatus });
 
 						// save all plan statuses into plan_status_change table
 						await PlanStatusChange.savePlanStatusChange(planStatuses);
+
+						// get all relevant mavat status for deposited plan
+						if (mavatStatus === null) {
+							const res = await PlanStatusChange.byMeirimStatus('התנגדויות והערות הציבור');
```

Comment:
chk status here correct @GalGend

Reply:
@drorgivton please approve this is the correct meirim status we should notify at
@yaelal please export to const

Reply:
@GalGend @yaelal confirmed - we should notify on ״התנגדויות והערות הציבור״

Reply:
@yaelal can you have consts for all fixed texts- of the statuses. You may do something like this in the Planstatus change file:

export const MEIRIM_STATUS = {
       DEPOSIT: 'הפקדה',
       PUBLIC_OBJECTION: 'הערות והתנגדויות הציבור',
        APPROVED: 'התוכנית אושרה'
};